src/test/java/utils/IntegrationTestSetupManager.java

```diff
@@ -31,9 +36,11 @@ public void createAdapterScript(final Statement statement, final String qualifie
     public void createTestTablesForJoinTests(final Statement statement, final String schemaName,
```

Comment:
This class completely breaks the SRP.
It is just a collection of methods that are needed somewhere else more than once.
When we split by dialect, we really need to clean this up.